standalone/jdaviz-cli-entrypoint.py

```diff
@@ -1,4 +1,5 @@
 import sys
+import pysiaf
```

Comment:
we'll need to decide what to do with this (and `hook-pysiaf`) given the plan to keep as a soft-dependency for now....

Reply:
What is this import for anyway? I don't see it used here.